### DISCUSSION TITLE
Trigger some code if the user tries to scroll past behind top / bottom content limit

### DISCUSSION THREAD

**Discussion by 90-008** [emilk/egui#1162] [Q&A]
Assuming I have a `ScrollArea`, and some content inside, what I want to do is trigger some code if the user tries to scroll past behind the bottom edge or the top edge. Is this possible? I have tried looking at how `ScrollArea` widget is implemented, but it's `State` isn't exposed to external consumers of the library; so I can't utilize it.

**Comment by emilk** [comment]
This hasn't been implemented it. Ideally `ScrollArea::show` would return a `struct` that contained a few flags indicating if the user tried to "overscroll" in either direction. It is likely not very hard to implement, though the code is a bit hairy :)
